### PR TITLE
Added workflow for pulling docs from api repo

### PR DESCRIPTION
**Pull request by adi790uu**
**What kind of change does this PR introduce?**

feature

**Issue Number:**

This PR is for the issue in api repo : PalisadoesFoundation/talawa-api/issues/1897

**Did you add tests for your changes?**

No

**Snapshots/Videos:**

**If relevant, did you update the documentation?**

No

**Summary**

This PR introduces a workflow which updated the talwa-api-repo folder to the latest changes.

**Does this PR introduce a breaking change?**

Yes

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-docs/blob/develop/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

File: .github/workflows/pull-latest-docs.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: Pull Talawa API Changes
+
+on:
+  schedule:
+    - cron: '*/5 * * * *'
```
Comment (KinetiCore-Labs) [inline .github/workflows/pull-latest-docs.yml]: Not every 5 minutes, but weekly. 
@palisadoes What is your suggestion? Daily, Weekly or Monthly.

File: .github/workflows/generate-docs.sh
Comment (KinetiCore-Labs) [inline .github/workflows/generate-docs.sh:1]: Is this script required?

File: .github/workflows/pull-docs.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+git clone --depth=1 --branch develop https://github.com/adi790uu/talawa-api.git
```
Comment (KinetiCore-Labs) [inline .github/workflows/pull-docs.sh]: Change the user.

File: .github/workflows/pull-latest-docs.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  pull:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout current repository
+        uses: actions/checkout@v2
+
+      - name: Set up Git config
+        run: |
+          git config --global user.email "${{ env.user_email }}"
```
Comment (KinetiCore-Labs) [inline .github/workflows/pull-latest-docs.yml]: env.email

**Review by KinetiCore-Labs — COMMENTED** [review]
Please address review comments.

File: .github/workflows/generate-docs.sh
Comment (adi790uu) [reply .github/workflows/generate-docs.sh:1]: @anwersayeed not really I can combine this script to pull-docs.sh but the purpose of this script is to generate docs and adjust its format did this just to separate some logic.

**Pull request closed without merging**